lib/menu.rb

```diff
@@ -0,0 +1,16 @@
+class Menu
+  attr_reader :name, :price
+
+  def initialize
+    @menu = []
+    @dish = Dish.new(@name, @price)
```

Comment:
Is this instance variable used anywhere?

Reply:
Thanks for picking that up. It is something I don't fully understand in my code. 

The feature and unit tests are failing if I change it to @dish, but I don't understand how the method `item` works on the method variable 'item' here

```  
def add(dish)
    @menu << dish.item
  end
```